index_en.html

```diff
@@ -333,9 +376,12 @@ <h2>Examples</h2>
           </pre>
       </aside>
       <p class="note">
-        Bullet Chatting make use of new components like
-        <code>bulletchatlist</code>和<code>bulletchat</code>，and they are not inherited from
-        <code>video</code> or <code>audio</code>.
+        Bullet Chatting uses new elements such as
+        <code>bulletchatlist</code> and <code>bulletchat</code>. These elements are not used as children of the media elements <code>audio</code> and <code>video</code>. From the Scenarios section, we can see that bullet chatting is not only related to media elements such as video. In scenes like Web page interaction or interactive wall, bullet chatting is independent to any media element.
```

Comment:
s/we can see that bullet chatting is not only related to media elements such as video/we can see that the usage of bullet chatting is not limited to those media elements like video

Reply:
Fixed: https://github.com/w3c/danmaku/pull/14/commits/def550f751355c4c16ffc31307e2d245f3a23788